src/gwt/src/org/rstudio/core/client/widget/FileOrUrlChooserTextBox.java

```diff
@@ -62,4 +96,53 @@ public String getText()
    {
       return getTextBox().getText();
    }
+   
+   @Override
+   public void onDetach()
+   {
+      checkTextBoxInterval_ = 0;
```

Comment:
Thanks! Double checked that this translates to `setTimeout` which only gets called once and for further calls we prevent using the `checkTextBoxInterval` flag.